### PR TITLE
Optimize release Docker Image

### PR DESCRIPTION
**Pull request by SuperSandro2000**
This PR combines some RUNs to reduce the layer count and also really delte the apt cache in all layers.

### CODE REVIEW

**Comment by nhooyr** [comment]
This was intended for readability and debugging. I don’t think it’s a good idea to always decrease layers, they exist for a reason.

**Comment by nhooyr** [comment]
There’s a bunch of old issues and PRs where this is discussed in detail. I do appreciate the pr @SuperSandro2000.

**Comment by nhooyr** [comment]
I’ll add some comments to clarify this for anyone in the future.

**Comment by SuperSandro2000** [comment]
Running ``RUN rm -rf /var/lib/apt/lists/*`` in its own layer rather then in the on ``apt-get update`` is run is useless and does not improve the overall download size in any way.

File: ci/release-image/Dockerfile [hunk 1]
```diff
@@ -1,20 +1,28 @@
...
-RUN locale-gen
+RUN sed -i "s/# en_US.UTF-8/en_US.UTF-8/" /etc/locale.gen \
+  && locale-gen
 ENV LANG=en_US.UTF-8
 
 RUN chsh -s /bin/bash
 ENV SHELL=/bin/bash
 
-RUN apt-get install -y dumb-init sudo
-RUN apt-get install -y man procps vim nano htop ssh git
```
Comment (SuperSandro2000) [inline ci/release-image/Dockerfile]: For example having these two installs right after each other does not improve anything and just adds another layer which decreases performance.

**Comment by nhooyr** [comment]
Yea in this specific case I was wrong, I'm sorry. The additional RUNs do not add to readability at all.

**Comment by nhooyr** [comment]
Thanks!

**Pull request merged**